lib/jekyll/converters/scss.rb

```diff
@@ -122,6 +122,11 @@ def convert(content)
       rescue ::Sass::SyntaxError => e
         raise SyntaxError.new("#{e.to_s} on line #{e.sass_line}")
       end
+
+      private
+      def site_source
+        @site_source ||= File.expand_path(@config["source"]).freeze
```

Comment:
On the one hand, that would mean you can pass any directory and it'll auto-expand, on the other `File.expand_path` calls are pretty expensive and we use this function a *lot*.